local/etc/pull_config.yaml

```diff
@@ -44,10 +44,10 @@
     - action: integrations
       branch: master
       globs:
-      - "**/metadata.csv"
-      - "**/manifest.json"
-      - "**/service_checks.json"
-      - "**/README.md"
+      - "*[!}]/metadata.csv"
+      - "*[!}]/manifest.json"
+      - "*[!}]/service_checks.json"
+      - "*[!}]/README.md"
```

Comment:
is this correct? i'm not familiar with `*[!}]`

Reply:
it was correct.. but too complicated for no reason. fixing it